scripts/retrieve_or_decode_configuration.py

```diff
@@ -0,0 +1,85 @@
+"""
```

Comment:
I think we'd want to update userdata.sh to use this script. Even though the config gets written directly there (through the template merge), it would make sense to be consistent: have the data.tf generate a base64, write it into userdata.sh which will call scripts/retrieve_or_decode_configuration.py.

WDUT?

Reply:
Done, as discussed we're going with the S3 route